web-api/src/business/useCases/courtIssuedDocument/fileAndServeCourtIssuedDocumentInteractor.ts

```diff
@@ -183,6 +183,9 @@ export const fileAndServeCourtIssuedDocument = async (
           { authorizedUser },
         );
 
+        docketEntryEntity.setDraftOrderState({
+          orderType: form.orderType,
+        });
```

Comment:
Why are we doing this here rather than in the constructor above? I don't necessarily think it is bad to have this function, but I do think it is probably worse to set `draftOrderState` in different ways. Now we have some places that do `draftOrderState: {` in the constructor and this place that does `setDraftOrderState`; it would be better to be consistent one way or the other. I would default to just using the constructor.